src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -1494,8 +1494,9 @@ private static void OnBreakEvent(Interop.Sys.CtrlCode ctrlCode)
                 handlerThread.Start(ctrlCode);
             }
 
-            private static void HandleBreakEvent(object state)
+            private static void HandleBreakEvent(object? state)
```

Comment:
the state arg for ControlCHandlerRegistrar.HandleBreakEvent had to match with target delegate ParameterizedThreadStart